Content.Shared/PDA/Ringer/RingerAccessUplinkComponent.cs

```diff
@@ -0,0 +1,24 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.PDA.Ringer;
+
+/// <summary>
+/// Opens the store UI when a PDA's ringtone is set to the secret code.
+/// Traitors are told the code when greeted.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class RingerAccessUplinkComponent : Component
```

Comment:
access. this also probably needs the autonetworking stuff and a sane API if you're setting the values frequently.